Jenkinsfile

```diff
@@ -5,19 +5,37 @@ pipeline {
             steps {
                 sh 'hatch run docs:build'
             }
+
+	    post {
+      		 success {
+        	  publishHTML([allowMissing: false,
+                     alwaysLinkToLastBuild: false,
+                     keepAll: true,
+                     reportDir: 'docs/site',
+                     reportFiles: 'index.html',
+                     includes: '**/*',
+                     reportName: 'Cycax Docs',
```

Comment:
To be pedantic the title is "CyCAx", all caps with small y and x. 
Docs >> docs
The line "reportTitles: 'Cycax Docs report'" seems weird, "Docs report". Lets just say "CyCAx docs"